### ISSUE TITLE
FS2, strange tensorboard outputs

### ISSUE THREAD

**Issue by ihshareef** [TensorSpeech/TensorFlowTTS#599]
I used Tacotron2 trained at 45k iterations (using teacher-forcing and window method) to extract durations for my dataset. I ran FastSpeech2 training without mixed precision. I turned off mixed precision because I was running into NaN issues.
When I checked my Tensorboard outputs at 115k iterations, I notice that the evaluation outputs don't look anything like it's supposed to.

![image](https://user-images.githubusercontent.com/7621646/122357606-55097480-cf6d-11eb-8c41-05292a207a63.png)

As expected, my trained FastSpeech2 (at 115k iterations) gives terrible results.
![image](https://user-images.githubusercontent.com/7621646/122360374-cba77180-cf6f-11eb-92e7-631169010e92.png)

I'm guessing that the core issue might have been in my Tacotron2 model and the duration extractions--mentioned in #597.
What could I do to fix this?

I'm using my custom-dataset, but it's formatted as per LJSpeech, and pre-processing seems to work fine. Are there any tests or checks that I can do during the duration-extraction phase to ensure that it had been done correctly?

**Comment by dathudeptrai** [comment]
@ihshareef I think the root caused is a bad duration extracted from Tacotron2, you may need training tacotron2 more. BTW, did you use our FS2 pretrained ?

**Comment by ihshareef** [comment]
You're right @dathudeptrai 
My Tacotron was poorly-trained, once I had fixed this ([as mentioned in another issue](https://github.com/TensorSpeech/TensorFlowTTS/issues/597#issuecomment-863810593)) I was able to properly train FS2 on my dataset and infer unseen sentences. I used a pretrained model (LJSpeech again) to fine-tune FS2, so I was able to get usable results (albeit with a gravelly voice on MB-MelGAN) within 5,000 iterations. I'm hoping that the gravelly voice goes away after more training.
